content/hi/docs/reference/glossary/app-container.md

```diff
@@ -4,7 +4,7 @@ id: app-container
 date: 2019-02-12
 full_link:
 short_description: >
-  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
+  कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर। इनिट (init) कंटेनर के साथ तुलना करें।
```

Comment:
I understand that this is just a reference doc. However can we start off by saying, @anubha-v-ardhan ?
```suggestion
ऐप कंटेनर कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर है। इनिट (init) कंटेनर के साथ तुलना करें।
```

Reply:
Can we write something along the lines of?:
```suggestion
 एक ऐसा कंटेनर जो कार्यभार के एक हिस्से को चलने के प्रयोग में किया जाता हैं। इनिट (init) कंटेनर के साथ तुलना करें।
```
WDYT, Thoughts? Thanks.

Reply:
I think @divya-mohan0209's suggestion `ऐप कंटेनर कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर है` sounds simpler.